pootle/core/utils/timezone.py

```diff
@@ -18,21 +18,42 @@
     datetime_min = timezone.make_aware(datetime_min, timezone.utc)
 
 
-def make_aware(value):
+def make_aware(value, tz=None):
+    """Makes a `datetime` timezone-aware.
+
+    :param value: `datetime` object to make timezone-aware.
+    :param tz: `tzinfo` object with the timezone information the given value
+        needs to be converted to. By default, site's own default timezone will
+        be used.
+    """
     if getattr(settings, 'USE_TZ', False) and timezone.is_naive(value):
-        tz = timezone.get_default_timezone()
-        value = timezone.make_aware(value, tz)
+        use_tz = tz if tz is not None else timezone.get_default_timezone()
+        value = timezone.make_aware(value, timezone=use_tz)
 
     return value
 
 
-def make_naive(value):
+def make_naive(value, tz=None):
+    """Makes a `datetime` naive, i.e. not aware of timezones.
+
+    :param value: `datetime` object to make timezone-aware.
+    :param tz: `tzinfo` object with the timezone information the given value
+        needs to be converted to. By default, site's own default timezone will
+        be used.
+    """
     if getattr(settings, 'USE_TZ', False) and timezone.is_aware(value):
-        tz = timezone.get_default_timezone()
-        value = timezone.make_naive(value, tz)
+        use_tz = tz if tz is not None else timezone.get_default_timezone()
+        value = timezone.make_naive(value, timezone=use_tz)
 
     return value
 
 
 def aware_datetime(*args, **kwargs):
-    return make_aware(datetime.datetime(*args, **kwargs))
```

Comment:
We are loosing the ability to pass the `kwargs`. Please pop out the tz first, and keep passing the kwargs to datetime.